ideascube/models.py

```diff
@@ -244,6 +244,14 @@ def index_strings(self):
         ('female', _('Female')),
     )
 
+    DISABILITY_CHOICES = (
+        ('visual', _('Visual')),
+        ('auditive', _('Auditive')),
+        ('physical', _('Physical')),
+        ('cognitive', _('Cognitive')),
+        ('mental', _('Mental')),
```

Comment:
Searching a bit, "intellectual disability" seems to be the preferred term in English, over "cognitive disability" and "mental disability" being the least preferred one.

These 3 seem to mostly be the same though, so maybe that's why Llave del saber didn't have the "mental" one?

In addition, the various categorizations seem to vary from one region to another, with sometimes "cognitive disability" being the same as "intellectual disability", but sometimes the former is a superset of the latter.

The more I research about this, the more I'm leaning towards a free-form `CharField`. It removes any assumption we could make about people with disabilities (assumptions are bad), and it makes sure we get it right by not excluding anybody.

It also removes the need to make annoying migrations in the future. For example, using a fixed set of categories, say a deployment wanted to split a category in two, we wouldn't be able to reassign people's disability to one or the other during the migration.